lib/Driver/ToolChains.cpp

```diff
@@ -1011,6 +1011,17 @@ static void getRuntimeLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
                           getPlatformNameForTriple(TC.getTriple()));
 }
 
+/// On Darwin we link fat libs in the root of the library path and on other OS
+/// targets we look up libraries in the target arch's subfolder.
+static void getRuntimeLibraryPathWithArch(SmallVectorImpl<char> &runtimeLibPath,
+                                  const llvm::opt::ArgList &args,
```

Comment:
Indentation seems off.

Reply:
yeah it's off on a few functions in this file it seems. Something a good clang-format/tidy pass might be good to do.